ipatests/test_ipaserver/test_install/test_bindinstance.py

```diff
@@ -33,6 +33,15 @@
 };
 
 include "random/file";
+
+dyndb "ipa" "/usr/lib/bind/ldap.so" {
```

Comment:
You have to add ``dyndb "ipa"`` to ``TEST_CONFIG``, too. The test uses the value of ``TEST_CONFIG`` as mocked data. Check out inbox.

Reply:
Haah, thanks!  I wasn't writing the data.  Adding the `flush()` was just because I thought maybe it wasn't being flushed.  Silly me, I did not update all the right data :)